src/OrleansAWSUtils/AWSUtilsSiloHostBuilderExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Orleans.Hosting;
+using OrleansAWSUtils.Configuration;
+
+namespace Orleans.Runtime.Hosting
```

Comment:
BTW, I believe we are sometimes adding to `Orleans.Runtime.Hosting` and sometimes to `Orleans.Hosting`. Let's be consistent. My opinion is that we go with where `ISiloHostBuilder` is, which is `Orleans.Hosting` as of today.